### ISSUE TITLE
Transfer native coin from BSC Testnet (BNB) to ICON testnet failed

### ISSUE THREAD

**Issue by FidelVe** [icon-project/icon-bridge#644]
## Overview
An attempt to transfer testnet BNB from BSC testnet chain to ICON testnet chain is failing.

## Steps to Reproduce
1. send X amount of BNB to [BTSCore logic contract on BSC testnet ](https://testnet.bscscan.com/address/0xe020d4ad483c7ec90a24d9db502e66564ef9c236#writeContract) writing the method `transferNativeCoin`

## Expected Behavior
Transaction should success

## Screenshots
![Screenshot from 2022-10-19 18-05-26](https://user-images.githubusercontent.com/43641727/196813590-e819e0eb-eb3f-4123-a5e5-215c98681fd4.png)

## Device Information

**Desktop (please complete the following information):**
 - OS: Ubuntu
 - Browser: N/A

## Additional Context

Is a fairly straightforward procedure, basically signing the method with the param `_to = btp://0x2.icon/hx0169e03001a3fa4012092ad4a4ddf2d07681f063` as you can see in the screenshot and the tracker tx the input was correct but the tx failed
Link to failed tx in the bsc tracker
https://testnet.bscscan.com/tx/0x0aa41b9f372eef3b4b8af13c0f3e125ee1d2e2be4018dba491b73d2829c74d2d

**Comment by han-so1omon** [comment]
@FezBox any ideas of who might know the answer here?

**Comment by FidelVe** [comment]
Trying to transfer doing the same method (`trasnferNativeCoin`) from ICON to BSC also gets reverted, here is the link for the fail tx

https://tracker.lisbon.icon.community/transaction/0x157a0bcd6625a0b1601b82c93e3b3b7971e76dd5f7e31e2ad9466a57199c6d2b

**Comment by FidelVe** [comment]
also trying to transfer btp-0x2.icon-bnUSD from ICON to BSC (testnet) fails.

here is the failed tx
https://tracker.lisbon.icon.community/transaction/0xe22aba72de91d1b4fa3b1f414ce0ec473df89527f27f754650dc724ff145f58c

here is an example of a success tx from ICON to BSC but they sent a different token, the data is the same format as my failed tx
https://tracker.lisbon.icon.community/transaction/0xa54eb0a7e756114fd4701ebb89b8b0dbee9de0bdbe13da44494a2f971da5f1fa

this is my failed tx data
```
{
	"method": "transfer",
	"params": {
		"_coinName": "btp-0x2.icon-bnUSD",
		"_to": "btp://0x61.bsc/0x4DeD312eB774B9828665448C55Faa8AE15353E56",
		"_value": "0x64"
	}
}
```

this is the data of the tx that was a success 6 days ago
```
{
	"method": "transfer",
	"params": {
		"_coinName": "btp-0x2.icon-sICX",
		"_to": "btp://0x61.bsc/0x2fF53aeC1Ac58b9691B22Be6cD8bad338b2F6ce8",
		"_value": "0x1158e460913d00000"
	}
}
```

we both had the same step limit so it should not be an issue related to the fee

**Comment by FezBox** [comment]
@manishbista28 Thoughts on the above?

**Comment by manishbista28** [comment]
Hi @FidelVe -- The amount that you have used to do the transaction was insufficient to cover the BTP fee charged in failed transfer cases and enough for the successful case.

For BSC-ICON transaction [0x0aa41b9f372eef3b4b8af13c0f3e125ee1d2e2be4018dba491b73d2829c74d2d](https://testnet.bscscan.com/tx/0x0aa41b9f372eef3b4b8af13c0f3e125ee1d2e2be4018dba491b73d2829c74d2d) the amount set was 0.001 BNB while the fixed fee charged for the native coin is 0.005 BNB.

Similar for ICON-BSC transaction [0xe22aba72de91d1b4fa3b1f414ce0ec473df89527f27f754650dc724ff145f58c](https://tracker.lisbon.icon.community/transaction/0xe22aba72de91d1b4fa3b1f414ce0ec473df89527f27f754650dc724ff145f58c), where a value of 100 was passed while fixed fee is 1500000000000000000.

You can query fee charged for different coins using [feeRatio method](https://tracker.lisbon.icon.community/contract/cxa843db0a27750230559f997bafaeb7f8739afc81#readcontract) on both chains.
![Screenshot from 2022-10-20 11-54-24](https://user-images.githubusercontent.com/66529584/196873420-d70268ea-5dcb-4802-9f64-cb6e20f96e03.png)
Fee calculation logic is present [here](https://github.com/icon-project/icon-bridge/blob/main/solidity/bts/contracts/BTSCore.sol#L417).

Additionally, we have run an instance of relay only to serve these [contract addresses](https://github.com/icon-project/icon-bridge/blob/main/docs/testnet_deployment.json).

Let me know if you have any query.

**Comment by CyrusVorwald-ICON** [comment]
Closed as resolved
